### PR TITLE
update docs: theme, text embeddings, used-by

### PR DESCRIPTION
**Pull request by tonyyanga**
Epsilla has a new cloud offering with a free tier. This PR mentions the cloud offering as an alternative way to running a docker container locally, which can be a big ask for some users.

A few other changes are explained in-line.

To see the updated version: https://trafilatura-epsilla.readthedocs.io/en/latest/tutorial-epsilla.html

### CODE REVIEW

File: docs/tutorial-epsilla.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Text embedding
+Tutorial: Text embedding
```
Comment (tonyyanga) [inline docs/tutorial-epsilla.rst:1]: Adding this title keeps it consistent with the other tutorials.

Before:
Screenshot 2023-11-24 at 7 22 10 PM

After:
Screenshot 2023-11-24 at 7 22 38 PM

File: docs/tutorial-epsilla.rst [hunk 2]
```diff
@@ -145,5 +151,7 @@ We can now perform a vector search to find the most relevant project based on a
 
 You will see the returned response is React! That is the correct answer. React is a modern frontend library, but PyTorch and Tensorflow are not.
 
+.. image:: https://static.scarf.sh/a.png?x-pxid=51f549d1-aabf-473c-b971-f8d9c3ac8ac5
```
Comment (tonyyanga) [inline docs/tutorial-epsilla.rst:154]: We are experimenting with [Scarf.sh](https://about.scarf.sh/documentation-insights). It is a privacy-friendly analytics product. Please let me know if you have any concerns about it.

**Comment by tonyyanga** [comment]
I have no idea why the CI tests are failing. They seem to suggest there are issues with the `test_dates` test case. It seems unrelated to the documentation updates.

**Comment by tonyyanga** [comment]
@adbar 

Looks like when htmldate is updated to 1.6.0, there are some behavior changes that impacted trafilatura tests. In this PR, I have bumped the required version for htmldate and updated the test cases for the new behavior.

See this colab notebook that demonstrates the changes in behavior: https://colab.research.google.com/drive/1UHiBcDlWt15qYlMZ6UKZG5ANs7ZRP0A0?usp=sharing

File: docs/tutorial-epsilla.rst [hunk 3]
```diff
@@ -28,7 +28,11 @@ In this tutorial, we will show you how to perform text embedding on results from
 Setup Epsilla
 ------------------------------------------------
 
-In this tutorial, we will run an Epsilla databse server. You can start one locally with a `Docker <https://docs.docker.com/get-started/>`_ image.
+In this tutorial, we will need an Epsilla databse server. There are two ways to get one: use the free cloud version or start one locally.
```
Comment (adbar) [inline docs/tutorial-epsilla.rst:31]: databse → database?

File: docs/tutorial-epsilla.rst [hunk 4]
```diff
@@ -28,7 +28,11 @@ In this tutorial, we will show you how to perform text embedding on results from
...
+
+Epsill has a `cloud version <https://cloud.epsilla.com//?ref=trafilatura>`_ with a free tier. You can sign up and get a server running in a few steps.
```
Comment (adbar) [inline docs/tutorial-epsilla.rst:33]: Epsill → Epsilla?

File: docs/tutorial-epsilla.rst (hunk 2 above)
Comment (adbar) [reply docs/tutorial-epsilla.rst:154]: Thanks for keeping me updated, I think it's fine.

**Comment by adbar** [comment]
Hi @tonyyanga, thanks for the changes, my remarks are above.

The change in htmldate is also tackled in #444 but I haven't merged it yet.

**Comment by tonyyanga** [comment]
Thanks for calling out the typos! Pushed a change to fix them.

**Pull request merged**